src/components/Editor/EditorInput/EditorInput.jsx

```diff
@@ -95,19 +97,26 @@ const EditorInput = () => {
   }, [project?.components]);
 
   useEffect(() => {
-    if (!project?.components) return;
+    const newFileNames = project.components.map(
+      (file) => `${file.name}.${file.extension}`,
+    );
+    if (newFileNames.join() !== fileNames?.join()) {
```

Comment:
Just checking - things won't be in a different order right?

Reply:
nice thinking and challenge. Just checking but it shouldn't matter. The only way we used to be able to alter the file order is through drag/drop. But the features currently not working and even if it was you'd probably want to scroll then anyway. This current approach scrolls for opened/reopened files but doesn't scroll for anything that doesn't change this array (i.e. `project.component.content` changes that currently invoke this on type)

Reply:
Roger! Makes sense